pypesto/objective/petab_import.py

```diff
@@ -93,10 +93,11 @@ def create_model(self, force_compile=False):
             exist yet. If True, the output folder is deleted and the model
             (re-)compiled in either case.
 
-
         .. warning::
             If `force_compile`, then an existing folder of that name will be
             deleted.
+
+        args, kwargs: Extra arguments passed to amici.SbmlImporter.sbml2amici
```

Comment:
These should belong in the parameters section.

Reply:
Hmm, shouldn't the warning be part of the respective parameter section?

Reply:
not sure if this can be interpreted correctly. maybe try to build the documentation using ``cd doc; make html; firefox _build/html/index.html`` and see how it looks.

Reply:
Thanks. Properly indented, but kept order.